src/tbb/global_control.cpp

```diff
@@ -48,6 +48,7 @@ class control_storage {
     std::size_t my_active_value{0};
     std::set<d1::global_control*, control_storage_comparator, tbb_allocator<d1::global_control*>> my_list{};
     spin_mutex my_list_mutex{};
+    virtual ~control_storage() = default;
```

Comment:
Let's move it to the `public` section alongside with others virtual methods.

Reply:
I agree. After approval I will squash the commits 🙂